### PR TITLE
Add neshanmap.ir, tapsell.ir, yektanet.com

### PR DESCRIPTION
**Pull request by keikhosrow**

### CODE REVIEW

File: data/category-ir-tech [hunk 1]
```diff
@@ -26,4 +27,6 @@ shatel.ir
 soft98.ir
 sotoon.ir
 taliya.ir
+tapsell.ir
```
Comment (IceCodeNew) [inline data/category-ir-tech:30]: if this domain belongs to an e-commerce site, we should move it to a category named `category-ir-e-commerce` or something.

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

File: data/category-ir-tech [hunk 2]
```diff
@@ -26,4 +27,6 @@ shatel.ir
...
 tci.ir
+yektanet.com
```
[the remark this answers is not in the document]
Comment (rootmelo92118) [reply data/category-ir-tech:32]: 截圖 2023-04-16 下午4 11 00
This website is about advertising.

File: data/category-ir-tech (hunk 1 above)
Comment (rootmelo92118) [reply data/category-ir-tech:30]: 截圖 2023-04-16 下午4 12 38
This website is also about advertising.

**Comment by IceCodeNew** [comment]
There are 3 actions we can take, which one would you prefer?
1. do not include `tapsell.ir` & `yektanet.com`
2. move these two domains into a category like `category-ir-ads`, which I consider the worst move. Blocking ads/trackers based on domains is not only inaccurate, but also a nightmare for reviewing and maintaining, and a fuse for the copyleft problems
3. keep these domains as they could be considered as _tech companies_

**Comment by rootmelo92118** [comment]
> There are 3 actions we can take, which one would you prefer?
> 
> 1. do not include `tapsell.ir` & `yektanet.com`
> 2. move these two domains into a category like `category-ir-ads`, which I consider the worst move. Blocking ads/trackers based on domains is not only inaccurate, but also a nightmare for reviewing and maintaining, and a fuse for the copyleft problems
> 3. keep these domains as they could be considered as _tech companies_
> 
> @rootmelo92118

@IceCodeNew I am prefer the third action. Do you have any opinion?

**Comment by keikhosrow** [comment]
Hello, due to the fact that the biggest advertising platform in Iran are four websites: Yektanet, Tapsel, Saba Vision and Tavus, I think the third option is better for now.

**Review by rootmelo92118 — APPROVED** [review]

**Comment by IceCodeNew** [comment]
> Hello, due to the fact that the biggest advertising platform in Iran are four websites: Yektanet, Tapsel, Saba Vision and Tavus, I think the third option is better for now.

I am OK with that. Thanks for feeding back.

**Review by IceCodeNew — APPROVED** [review]

**Pull request merged**